applications/app/services/NewspaperQuery.scala

```diff
@@ -22,26 +22,32 @@ object NewspaperQuery extends ExecutionContexts with Dates with Logging {
 
   val dateForFrontPagePattern = DateTimeFormat.forPattern("EEEE d MMMM y")
   val FRONT_PAGE_DISPLAY_NAME = "front page"
+  val pathToTag = Map("theguardian" -> "theguardian/mainsection", "theobserver" -> "theobserver/news")
 
-  def fetchTodaysPaper: Future[List[FaciaContainer]] =
-    bookSectionContainers(DateTime.now(DateTimeZone.UTC))
+  def fetchLatestGuardianNewspaper: Future[List[FaciaContainer]] = {
+    val date = DateTime.now(DateTimeZone.UTC)
+    bookSectionContainers("theguardian/mainsection", getLatestGuardianPageFor(date))
+  }
 
+  def fetchLatestObserverNewspaper: Future[List[FaciaContainer]] = {
+    val now = DateTime.now(DateTimeZone.UTC)
```

Comment:
Teeny niggle - might we use `val date = ...` here or `val now = ...` on line 28?